kong/conf_loader/constants.lua

```diff
@@ -499,6 +499,7 @@ local CONF_PARSERS = {
   cluster_use_proxy = { typ = "boolean" },
   cluster_dp_labels = { typ = "array" },
   cluster_rpc = { typ = "boolean" },
+  cluster_cjson = { typ = "boolean" },
```

Comment:
How is this option used?

Reply:
It is a "hiden" option, like "legacy_events" "legacy_dns_client"

Reply:
Seems I cannot find assignment to this var in this PR.

Reply:
It is a env var.